[EDU-6019] Remove RTM | Add console steps to clone edge application guide (#1508)
aziontech/docs: src/content/docs/en/pages/guides/edge-application/ea-clone-edge-application.mdx

To clone an existing edge application, follow these steps: 

1. Access [Azion Console](/en/documentation/products/guides/how-to-access-azion-console/) > **Edge Application**.
    - You'll be redirected to the **Edge Application** page, which lists all the edge applications you've created. 
    - The list is organized alphabetically. You can also use the **Search** bar located on the upper-left corner of the list.
2. Find the edge application you want to clone.
3. Click the three horizontal dots in the right-corner.
4. Select the **Clone** option.
5. Type a unique name for your new Edge Aplication in the popover.

> - Type a unique and easy to remember name.

Reviewer: ```suggestion
    - Type a unique and easy to remember name.
```

> 5. Type a unique name for your new Edge Aplication in the popover.

Reviewer 2: ```suggestion
5. Type a unique name for your new edge application in the popover.
```

    - The platform returns an error message if the name is already in use. If that happens, try again with a different name.
6. Click the **Clone** button.

> 7. You'll be redirected to the **Edit Edge Application** page. Click the **Save** button to save your Edge Applications.

Reviewer 2: ```suggestion
7. You'll be redirected to the **Edit Edge Application** page. Click the **Save** button to save the configurations for your edge application.
```
